projects/packages/videopress/src/client/block-editor/global.d.ts

```diff
@@ -7,6 +7,8 @@ declare global {
 		videoPressEditorState: {
 			extensions: VideoPressExtensionsProps;
 			siteType: 'simple' | 'atomic' | 'jetpack';
+			adminUrl: string;
```

Comment:
Are we setting/using this variable somewhere?

Reply:
I think not, it was added [here](https://github.com/Automattic/jetpack/pull/28501/commits/3ac36d5a32c7d52c8b589cc43e4ab35c281eb7bf) but not used

Reply:
It's also not being set anymore, so I think we can drop it for now.

```suggestion
```